### ISSUE TITLE
question about the creation of environment from saved checkpoint

### ISSUE THREAD

**Issue by yanan1116** [ARISE-Initiative/robomimic#281]
hi authors,

I  trained my VLA model based on the officially provided collected trajectories (https://huggingface.co/datasets/amandlek/robomimic/tree/main/v1.5) which works fine in `run_trained_agent.py`, with success rate higher than 90%.

I have a question about this part, which creates environment from saved checkpoint
https://github.com/ARISE-Initiative/robomimic/blob/master/robomimic/scripts/run_trained_agent.py#L204

It seems that the `env` comes from the saved checkpoint , like `lift_ph_low_dim_epoch_1000_succ_100.pth`.

So, is the `env` exactly or quite similar to what the model was facing during the training ? if that is the case, this high SR could indicate overfit ?

the second question is, how to modify the `env` from `FileUtils.env_from_checkpoint` to inject some randomization such that the testing can be more different from the original `env` . 

thanks.

**Comment by amandlek** [comment]
The environment will have the same kwargs but every env.reset call should generate a new task instance that is from the same reset distribution, but not exactly the same as those in the training dataset. You could change the seed in run_trained_agent.py to further promote more randomization, or you could change the environment name to use a modified environment with different properties.
